src/main/java/com/javagirls/MangoRoom/service/ReservationService.java

```diff
@@ -1,21 +1,24 @@
 package com.javagirls.MangoRoom.service;
 
 import com.javagirls.MangoRoom.dto.ReservationDto;
-import com.javagirls.MangoRoom.dto.RoomDto;
 import com.javagirls.MangoRoom.entity.Reservation;
 import com.javagirls.MangoRoom.entity.Room;
 import com.javagirls.MangoRoom.mapper.ReservationMapper;
 import com.javagirls.MangoRoom.repository.ReservationRepository;
+import com.javagirls.MangoRoom.repository.RoomRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.Optional;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
 
 @Service
 public class ReservationService {
 
     private ReservationRepository reservationRepository;
     private ReservationMapper mapper;
+    private RoomRepository roomRepository;
```

Comment:
jeśli dodajesz nowe dependency to musisz dodać je również w konstruktorze. IntelliJ nie podkreślił?? 
ewentualnie proponuje użycie @AllArgsConstruktor

Reply:
myślałam o tym, ale widziałam, że mapper wcześniej nie był dodany, więc pomyślałam, że tak ma być

Reply:
powinno być wszystko dodane